src/components/new-request/search.vue

```diff
@@ -462,11 +478,18 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
       if (this.getEntityTypeCd && this.isNamedCompany && !this.isFederal) return true
     }
 
+    // Conditional for Change Name Flow.
     if (this.isChangeName) {
       if (this.getEntityTypeCd && this.isNamedCompany && !this.isFederal) return true
       if (this.getEntityTypeCd && this.isSociety) return true
     }
 
+    // Conditional for Continuation In Flow.
+    if (this.isContinuationIn) {
+      if (this.getEntityTypeCd && this.isNamedCompany) return true
+      if (this.getEntityTypeCd && this.isSociety) return true
+    }
```

Comment:
Test this with Coop also -- it's not a numbered entity type so the user didn't select named company.

Reply:
Yes. I found while testing and it is fixed.